src/routes/projects.tsx

```diff
@@ -23,13 +28,33 @@ export const Route = createFileRoute("/projects")({ component: ProjectsIndexRout
  */
 function ProjectsIndexRoute() {
   const { t } = useTranslation();
+  const { projectsApi } = useApi();
+  const queryClient = useQueryClient();
+  const showConfirmDialog = useConfirmDialog();
 
   const [paginationModel, setPaginationModel] = useState<GridPaginationModel>({ page: 0, pageSize: 10 });
   const [first, max] = usePaginationToFirstAndMax(paginationModel);
   const listProjectsQuery = useListProjectsQuery({ first, max });
   const maxResults = useCachedMaxResultsFromQuery(listProjectsQuery);
   const projects = listProjectsQuery.data?.projects;
 
+  const deleteProjectMutation = useMutation({
```

Comment:
Since we added docs to some other mutation utilizing functions - lets add docs here as well :D